### DISCUSSION TITLE
Feedback sobre o SisLT

### DISCUSSION THREAD

**Discussion by gloriadeitos** [CEPAG-UFPR/Discussions#5] [SisLT]
<p align="center">
  <img src="https://campusmap.ufpr.br/dev/cepag-modelos/img/parcerias/aerosat.png" alt="AEROSAT Logo" width="70">
  <img src="https://campusmap.ufpr.br/dev/cepag-modelos/img/cepag/cepag.png" alt="CEPAG Logo" width="100">
</p>

Esta é a seção dedicada a receber o feedback sobre o **SisLT**. Se você tiver sugestões, críticas ou qualquer comentário, sinta-se à vontade para compartilhá-los aqui!

Sua opinião é muito importante para nós e ajudará a melhorar o projeto. Lembre-se de que, para participar da discussão, você precisará de uma conta do GitHub.

**Agradecemos sua colaboração e estamos ansiosos para ouvir suas ideias!**
**Atenciosamente, Equipe do CEPAG**

<table  align="center">
  <tr>
    <td align="center" valign="top" width="80"><br />
      <a href="https://youtu.be/b0e8140kZRc">
        <img src="https://campusmap.ufpr.br/dev/cepag-modelos/img/icones/youtube.png" />
      </a>
    </td>
    <td valign="top">
      <h3>Como dar feedback no GitHub</h3>
      <p>Um vídeo explicando como deixar comentários no GitHub.</p>
      <a href="https://youtu.be/ymNI24EyOL4">
        <img src="https://img.shields.io/badge/Ver%20Video-FF0000?style=for-the-badge" target="_blank" alt="Ver Video">
      </a>
    </td>
  </tr>
</table>

**Comment by rhfelipe** [comment]
### Nuvem RGB e classificada

Olá, como conversado na última reunião adicionar as nuvens RGB e classificada para vermos como ficará na visualização. Os arquivos .laz com as respectivas nuvens estão no drive abaixo compartilhado com a Maíra.

https://drive.google.com/drive/folders/1LFq9LA8Sk-299qYtAM-8tERBacBCihdX

**Reply by MairaTC** [reply]
Recebido @rhfelipe, vou adicionar estas nuvens ao sistema

**Comment by lefho1906** [comment]
@rhfelipe informamos que o arquivo "LAZ_F22 - 241015_190450_VQ_480_02_RGB-classificado.laz" foi carregado pela @MairaTC no Potree e está disponível para visualização na linha **Campo Largo - Linha2**. Solicitamos verificar as opções de visualização de cor RGB, intensidade de retorno, coloração por elevação e classificação e retornar. Obrigado
